src/javaserver/config/RoutesConfig.java

```diff
@@ -0,0 +1,12 @@
+package javaserver.config;
+
+import javaserver.Routes.RoutesRegistrar;
+
+public class RoutesConfig {
+    public static void seedRoutes() {
```

Comment:
I would look at extracting a private method for these calls. Either just the `RoutesRegistrar.getInstance()` part or the entire call to `RoutesRegistrar.getInstance().registerRoute`.